src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -173,33 +173,31 @@ define(function (require, exports, module) {
             stat(path, callback);
         });
     }
-
-    function rename(oldPath, newPath, callback) {
+   function rename(oldPath, newPath, callback) {
         oldPath = decodePath(oldPath);
         newPath = decodePath(newPath);
 
         function updateBlobURL(err) {
             if(err) {
                 return callback(_mapError(err));
             }
-
+        fs.refresh(callback);
+        fs.reload();
```

Comment:
These two lines are happening in the wrong place.  You likely want to do this down around 196 below.